api/docker/Dockerfile

```diff
@@ -4,12 +4,21 @@ RUN apt-get update
 
 RUN apt-get install -y \
     pkg-config  \
-    libzip-dev
+    libzip-dev \
+    iproute2 \
+    libpcre3-dev \
+    supervisor \
+    vim
 
 RUN docker-php-ext-install \
     mysqli \
     pdo_mysql \
-    zip
+    zip \
+    pcntl \
+    sockets
+
+RUN pecl install redis
+RUN docker-php-ext-enable redis pcntl sockets
```

Comment:
Thanks for catching this oversight, we told you redis was available via the compose project but we didn't install the php extension for it.